exasol_script_languages_developer_sandbox/lib/setup_ec2/run_setup_ec2.py

```diff
@@ -16,10 +16,25 @@
 
 LOG = get_status_logger(LogType.SETUP)
 
+# Python version of a typedef: Defines a type for the iterator we use for the state machine
+EC2LifecycleDataIterator = Iterator[Tuple[Optional[EC2Instance], Optional[str]]]
```

Comment:
Maybe create a typedef also for the tuple, this probably makes this typedef easier to read